templates/components/common/quick-search.html

```diff
@@ -1,11 +1,11 @@
 <div class="js-container">
     <form onsubmit="return false" data-url="{{urls.search}}" class="js-quick-search-form">
-        <div class="js-form-field">
+        <div class="js-form-field c-quick-search">
             <label class="u-hidden-visually" for="{{name}}">
                 {{lang "search.quick_search.input_label"}}
             </label>
             <input 
-                class="js-form-input js-search-quick"
+                class="js-form-input js-search-quick c-form__input"
```

Comment:
maybe `o-form__input c-quick-search__input` ?  The `c-form__input` seems like it doesn't belong here